### DISCUSSION TITLE
Community driven index of Jupiter extensions?

### DISCUSSION THREAD

**Discussion by Bananeweizen** [junit-team/junit-framework#2851] [Q&A] [answered]
I'm somewhat certain I once stumbled over an extensive list of Jupiter extensions with more than 50 entries. I have not been able to find it again via google, stackoverflow and github search. Is anyone aware of such a thing? If not, should something like that be part of the JUnit documentation or be maintained in a separate repository as simple readme with pull requests?

My main motivation is: "I need something like TempDirectory, but with additional foobar functionality. Is it in JUnit Pioneer, in BigFrameworkXYZ JUnit Extension or do I have to write my own?"

**Comment by jbduncan** [comment, marked as answer]
I believe you're after the wiki page on [Third-party Extensions](https://github.com/junit-team/junit5/wiki/Third-party-Extensions). :)

**Comment by sormuras** [comment]
What @jbduncan said, especially the "**JUnit Jupiter Extensions**" on that wiki page.
